openedx/features/course_experience/templates/course_experience/dates-summary.html

```diff
@@ -0,0 +1,26 @@
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%page args="course_date" expression_filter="h"/>
+<div class="date-summary-container">
+    <div class="date-summary date-summary-${course_date.css_class}">
+        % if course_date.title:
+            % if course_date.title == 'current_datetime':
+                <span class="hd hd-6 heading localized-datetime" data-datetime="${course_date.date}" data-string="${_(u'Today is {date}')}" data-timezone="${user_timezone}" data-language="${user_language}"></span>
+            % else:
+                <span class="hd hd-6 heading">${course_date.title}</span>
+            % endif
+        % endif
+        % if course_date.date and course_date.title != 'current_datetime':
+            <p class="hd hd-6 date localized-datetime" data-format="shortDate" data-datetime="${course_date.date}" data-timezone="${user_timezone}" data-language="${user_language}" data-string="${_(course_date.relative_datestring)}"></p>
+        % endif
+        % if course_date.description:
+            <p class="description">${course_date.description}</p>
```

Comment:
This should be translated as well

Reply:
@ahsan-ul-haq I don't think marking this for translation will work, how translator will translate a variable value which they don't know?